src/esp/gfx/DrawableGroup.h

```diff
@@ -3,10 +3,14 @@
 // LICENSE file in the root directory of this source tree.
 
 #pragma once
+
 #include <Magnum/SceneGraph/Drawable.h>
 #include <Magnum/SceneGraph/FeatureGroup.h>
+#include <Magnum/SceneGraph/SceneGraph.h>
 #include <unordered_map>
 
+#include <functional>
+#include <unordered_map>
```

Comment:
isn't clang tidy complaining here? the include is used twice

Reply:
I do not have clang tidy installed locally on a linux machine. Will do it.